Apromore-Custom-Plugins/User-Admin-Portal-Plugin/src/main/java/org/apromore/plugin/portal/useradmin/UserAdminController.java

```diff
@@ -1165,7 +1165,7 @@ public void onClickAssignUser() {
                 nonAssignedUserList.reset();
                 assignedUserList.reset();
             }
-            isGroupDetailDirty = false;
+            isGroupDetailDirty = true;
```

Comment:
The naming "isGroupDetailDirty" is quite impressive, but a more meaningful name would be "isGroupDetailChanged", as I can understand it easily.

Reply:
I think "dirty" flag is common term to indicate unsaved changes in many JS libraries.
https://github.com/ngneat/dirty-check-forms
https://www.npmjs.com/package/jquery.dirty